mobile/lib/main_common.dart

```diff
@@ -29,6 +29,18 @@ class AirQoApp extends StatelessWidget {
         BlocProvider(
           create: (BuildContext context) => NearbyLocationBloc(),
         ),
+        BlocProvider(
+          create: (BuildContext context) => SettingsBloc(),
+        ),
+        BlocProvider(
+          create: (BuildContext context) => AuthCodeBloc(),
+        ),
+        BlocProvider(
+          create: (BuildContext context) => PhoneAuthBloc(),
+        ),
+        BlocProvider(
+          create: (BuildContext context) => EmailAuthBloc(),
+        ),
```

Comment:
Dependency injection to enable usage of a single bloc instance in multiple widgets.